Jekyll is once again emitting a putrescent stench

```
Run gem install bundler
Successfully installed bundler-2.2.5
Parsing documentation for bundler-2.2.5
Installing ri documentation for bundler-2.2.5
Done installing documentation for bundler after 5 seconds
1 gem installed
Fetching gem metadata from http://rubygems.org/.........
listen-3.2.1 requires ruby version >= 2.2.7, ~> 2.2, which is incompatible with
the current version, ruby 3.0.0p0
Error: Process completed with exit code 5.
```

Builds that worked last week are now all failing like that.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -53,6 +53,8 @@ jobs:
           key: ${{ runner.os }}-sbt-cache-v2-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
 
       - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7
 
       - name: Install Jekyll
         run: |

--- a/build.sbt
+++ b/build.sbt
@@ -152,7 +152,7 @@ inThisBuild(List(
   githubWorkflowTargetTags ++= Seq("v*"),
   githubWorkflowPublishTargetBranches := Seq(RefPredicate.StartsWith(Ref.Tag("v"))),
   githubWorkflowBuildPreamble ++= Seq(
-    WorkflowStep.Use("actions", "setup-ruby", "v1"),
+    WorkflowStep.Use("actions", "setup-ruby", "v1", params = Map("ruby-version" -> "2.7")),
     WorkflowStep.Run(List(
       "gem install bundler",
       "bundle install --gemfile=site/Gemfile"

